Lib/gflanguages/data/languages/ltg_Latn.textproto

```diff
@@ -8,5 +8,5 @@ region: "LV"
 exemplar_chars {
   base: "A Ā B C Č D E Ē F G Ģ H I Ī J K Ķ L Ļ M N Ņ O Ō P R S Š T U Ū V Y Z Ž a ā b c č d e ē f g ģ h i ī j k ķ l ļ m n ņ o ō p r s š t u ū v y z ž"
   auxiliary: "Q W X q w x"
-  marks: "◌̄ ◌̌ ◌̧"
+  marks: "◌̄ ◌̌ ◌̧ ◌ ̒"
```

Comment:
Adding 0x0312 COMBINING TURNED COMMA ABOVE to Latvian (lv), Latgalian (ltg), Prussian (prg) is incorrect. While the glyph may be used as a component of ģ, the character is not used in those languages.